components/src/molecules/cookie-message/CookieMessage.scss

```diff
@@ -74,34 +78,36 @@ $cookie-message__close-icon-transition : $transition-base !default;
     cursor: pointer;
 
     &:active,
-    &:hover {
+    &:hover,
+    &:focus {
       color: $cookie-message__close-color;
       background: none;
+
+      .a-cookie-message__close-icon {
+        transition: $cookie-message__close-icon-transition;
```

Comment:
Move this transition to `&__close-icon` styles, now the transition effect works only in one-way